index.html

```diff
@@ -0,0 +1,571 @@
+```
+
+A common use case is that of a processor which consumes a valid input schema and generates an output schema.
+
+The general guidance for processor behavior is: don't react to what you don't understand.
+
+Specifically, processors:
+  - **should** pass through `@core` directives which reference unknown feature URLs
+  - **should** pass through prefixed directives, types, and other schema elements  
+
+An exception to this is processors which prepare the schema for final public consumption. Such processors **may** choose to eliminate all unknown directives and prefixed types in order to hide schema implementation details within the published schema. This will impair the operation of tooling which relies on these directives—such tools will not be able to run on the output schema, so the benefits and costs of this kind of information hiding should be weighed carefully on a case-by-case basis.
+
+## Validations &amp; Algorithms
+
+### Bootstrapping *validates* Has Core Feature
+
+Determine the name of the core specification within the document *or* fail the *Has Core Feature* validation.
+
+It is possible to [rename the core feature](#renaming-core-itself) within a document. This process determines the actual name for the core feature if one is present.
+
+**Formal Specification**
+- For each directive `d` on the SchemaDefinition within the Document,
+  - If `d` has a `feature:` argument whose value is "https://lib.apollo.dev/core/v0.1", *and either*:
+    - &#8230;`d` has an `as:` argument whose value is equal to `d`'s name
+    - &#8230;*or* `d` does not have an `as:` argument and `d`'s name is `core`
+    - *then* **Return** `coreName` = `d`'s name
+- If no matching directive was found, the ***Has Core Feature* validation fails**.
+
+### Namespace Collection *validates* Name Uniqueness
+
+Collect a map of `featureName: String -> Directive`, where `Directive` is a `@core(feature:)` Directive which introduces some feature named `featureName` into the document.
+
+**Formal Specification**
+- Given [`coreName`](#bootstrapping-validates-has-core-feature)
+- Let `namespaces` be a map of `featureName: String -> Directive`, initially empty.
+- For each directive `d` named `coreName` on the SchemaDefinition within the Document,
+  - Let `name` be the spec's [name](#prefixing) as specified by the directive's `as:` argument or, if the argument is not present, the default name specified in the [feature url](#feature-urls).
+  - If `name` exists within `namespaces`, the ***Name Uniqueness* validation fails**.
+  - Insert `name => d` into `namespaces`
+- **Return** `namespaces`
+
+**Explanatory Text**
+
+Prefixes, whether implicit or explicit, must be unique within a document. Valid:
+
+<view-of src=prefixing.graphql#schema[0]
+  title="Example of unique prefixes"
+></view-of>
+
+It is also valid to reference multiple versions of the same spec under different prefixes:
+
+<view-of src=prefix-uniqueness.graphql#schema[0]
```

Comment:
The `#schema[0]` thing does not appear to be working for me. I'm seeing the full prefix-uniqueness file inserted every time.